aws_lambda_builders/workflows/nodejs_npm/actions.py

```diff
@@ -151,7 +159,11 @@ def execute(self):
         try:
             LOG.debug("NODEJS installing ci in: %s", self.install_dir)
 
-            self.subprocess_npm.run(["ci"], cwd=self.install_dir)
+            command = ["ci"]
+            if self.install_links:
+                command.append("--install-links")
```

Comment:
something to keep in mind, is that we are appending in certain places and doing a + operation in other places, let's default to one.

Reply:
I don't think there's a problem with using both 🤔 . Appending to add one element, using "+" to concatenate two lists.

Reply:
I don't have any strong opinion here, but we can also use `.extend()` to concatenate two lists if we feel that's cleaner moving forward, though I think `+=` is usually descriptive enough.